src/Akeneo/Connectivity/Connection/workspaces/permission-form/src/registry/PermissionFormRegistry.ts

```diff
@@ -41,6 +41,11 @@ const PermissionFormRegistry = {
             })
         );
     },
+    countProviders: (): number => {
```

Comment:
```suggestion
    count: (): number => {
```
I would go for a simpler method name.

Reply:
why ? I'd rather be explicit when I name something to avoid confusion.